easybuild/easyconfigs/d/DL_POLY_Classic/DL_POLY_Classic-1.9-foss-2015b-PLUMED-2.1.4.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'MakeCp'
```

Comment:
In this case, the easyblock is reinventing the `MakeCp` block. 

Yes, there is duplicate information (as is everywhere) but yaml is the solution to that, not an easyblock.